OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -71,7 +71,8 @@ MANAGED_SERVER_PORT=<Managed Server Port>
 CONFIGURE_UCM_CONNECTION=<true or false>
 #Valid option for socket type are socket,jaxws 
 UCM_SOCKET_TYPE=<UCM Socket Type >
-#Required if UCM_SOCKET_TYPE is jaxws
+#If UCM_SOCKET_TYPE is socket configure UCM_HOST=(hostname or IPAddress) of UCM machine
+#If UCM_SOCKET_TYPE is jaxws configure UCM_HOST=(protocol://hostname or IPAddress) of UCM machine
 UCM_HOST=<UCM Host>
```

Comment:
remove above two comments and instead use the one below
# Set to true if UCM is using SSL
UCM_USING_SSL=false